c++/src/capnp/raw-schema.h

```diff
@@ -28,6 +28,26 @@
 
 #include "common.h"  // for uint and friends
 
+#if _MSC_VER
+
+// TODO(msvc):  RawSchema and RawBrandedSchema use some atomic operations in their implementations.
+// The only non-deprecated atomic operations that MSVC supports is std::atomic. We could rewrite our
+// atomic operations to use std::atomic, but this has the awkward consequence of making RawSchema
+// and RawBrandedSchema non-POD, non-trivially-copyable, non-standard-layout, etc. It is likely safe
+// to ignore this consequence -- i.e., declare the atomic variables as Initializer* and cast them to
+// std::atomic<Initializer*> as necessary -- but clearly non-portable beyond MSVC, which defeats the
+// purpose of using std::atomic. Another, perhaps better, option would be to use MSVC's old
+// deprecated operations, which are unlikely to disappear any time soon.
+//
+// Since Cap'n Proto code is required to be single-threaded anyway for the foreseeable future, we
```

Comment:
Thanks for explaining this, the need is clear to me now.

My hesitance to use the deprecated API is due in large part to lack of familiarity: I never touched an atomic operation until a post-C++11 world, and I don't want to get it wrong. However, given our options, they seem least bad, so I'll give it a shot. :) I'll take a look at `kj::Once` first, though, as it'd be nice to factor the platform-dependent code out somewhere else.